### ISSUE TITLE
Hue Beyond - LLM001

### ISSUE THREAD

**Issue by rickardp** [sebbebebbe/com.philips.hue.zigbee#345]
I reported issues earlier with this lamp (#25) after which I switched to using the Hue bridge. Now that Homey did work on Zigbee I decided to give this another shot. I see traces in your code of support for this lamp, but I'm not getting this lamp to pair.

After resetting it using the Hue remote, I can only pair a "Basic zigbee device" with product ID LLM001. The basic device works, but can only dim either the top or bottom part (seemingly at random when I pair, at least I failed to see the pattern). After changing the driver.compose.json to read `"productId": ["HBL001", "LLM001"],`, I get the device to pair as a Beyond device, but I still can only dim one part of the lamp (turning off turns off only the part that dims currently the bottom part).

Is the idea that this lamp is supported? If it is, maybe there exists different hardware revisions of it? I'm happy to assist with info in this case. If the intent is to not support it, maybe at least update the docs to say so would be a good idea, currently it is documented as working, both here on Github and on the Homey app store.

**Comment by rickardp** [comment]
Some additional info after playing around a bit more with pairing. I actually got both parts to pair! I had to pair once, then power cycle then pair again. Now with my "patched" LLM001 app, I got each part of the lamp to show up as a Beyond lamp of its own. Looks like possibly Philips implemented the lamp as two independent Hue lamps, complete with their own Zigbee IDs (they even show different MAC addresses).

Maybe adding the light module LLM001 as a generic Hue light module would help?

I managed to dig out that my lamp is hw revision 6 and firmware 5.130.1.30000 in case there are multiple different behaviours.
